<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuousResultsWindow</class>
 <widget class="QWidget" name="ContinuousResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="loadFromCSV">
     <property name="text">
      <string>Nahraj v√Ωsledky z .CSV souboru (generuje excel)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="resultsTableV"/>
   </item>
   <item>
    <widget class="QPushButton" name="generateHtmlBtn">
     <property name="text">
      <string>html</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="htmlPText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
